### ISSUE TITLE
Exception when updating Boxstarter.Azure from 2.4.159 to 2.4.180

### ISSUE THREAD

**Issue by TimMurphy** [mwrock/boxstarter#63]
When running `choco update all` or `choco update Boxstarter.Azure` I receive the following exception report.

Running Windows 8.1 with Administrator command prompt.

```
Updating Boxstarter.Azure from 2.4.159 to 2.4.180
Chocolatey (v0.9.8.28) is installing 'Boxstarter.Azure' and dependencies. By installing you accept the license for 'Boxstarter.Azure' and each dependency you are installing.

Boxstarter.Azure v2.4.180
Write-Error : Boxstarter.Azure did not finish successfully. Boo to the chocolatey gods!
-----------------------
[ERROR] Could not load file or assembly '.\/boxstarter.Azure.psm1' or one of its dependencies. The given assembly name or codebase was invalid. (Exception from HRESULT: 0x80131047)
-----------------------
At C:\ProgramData\chocolatey\chocolateyinstall\helpers\functions\Write-ChocolateyFailure.ps1:30 char:3
+   Write-Error $errorMessage
+   ~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,Write-Error

Write-Error : Package 'Boxstarter.Azure v2.4.180' did not install successfully: Could not load file or assembly '.\/boxstarter.Azure.psm1' or one of its dependencies. The given assembly name or codebase was invalid. (Exception from HRESULT: 0x80131047)
At C:\ProgramData\chocolatey\chocolateyinstall\functions\Chocolatey-NuGet.ps1:90 char:17
+                 Write-Error "Package `'$installedPackageName v$installedPackageV ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,Write-Error

Finished installing 'Boxstarter.Azure' and dependencies - if errors not shown in console, none detected. Check log for errors if unsure.

C:\Windows\system32>
```

**Comment by mwrock** [comment]
just released a fix in v2.4.183.

Thanks for reporting this!

**Comment by TimMurphy** [comment]
Just installed without issue. Thanks for the fix and magnificent product.
